src/mumble/ConnectedQuitBehaviour.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_CONNECTEDQUITBEHAVIOUR_H
+#define MUMBLE_CONNECTEDQUITBEHAVIOUR_H
+
+#include <QtCore/QObject>
```

Comment:
Never include Qt headers from "sub-directories". This is discouraged. Always prefer the global include
```suggestion
#include <QObject>
```

However, I would even argue that we don't need this include at all - at least not in the header :eyes: